mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -4915,9 +4935,8 @@ struct OrderForRpcWithCancellationReason<'a> {
 
 pub async fn order_status(ctx: MmArc, req: Json) -> Result<Response<Vec<u8>>, String> {
     let req: OrderStatusReq = try_s!(json::from_value(req));
-
+    let db_id: Option<String> = None; // TODO
```

Comment:
This is tricky and not easy to solve, we need to search for a uuid across all databases. We need to check how this is used in GUIs if we want to simplify it.

Reply:
For this, I see now way but to have active `db_id`s saved in context and accessed one by one starting with none in a loop here https://github.com/KomodoPlatform/komodo-defi-framework/blob/4b99dbad136dbfb60d1805d8a391aff5f7ef33f3/mm2src/mm2_main/src/lp_ordermatch.rs#L4965-L4978